### DISCUSSION TITLE
Artifacts UI & RAG broken for Claude

### DISCUSSION THREAD

**Discussion by baalajimaestro** [danny-avila/LibreChat#3846] [Troubleshooting] [answered]
### What happened?

I use a proxy to claude api. I suppose librechat is sending system instructions to anthropic api as an array, like how it would to OpenAI. Tried the same with OpenAI GPT-4o and it works pretty fine.

```
{
  "error": {
    "message": "Request validation failed. At 'system': Expected string, received array",
    "type": "proxy_validation_error"
  }
}
```

Update: RAG also appears to be broken with the same error message. Similarly, it works on GPT-4o.

### Steps to Reproduce

1. Enable Artifacts UI from Beta
2. Use claude sonnet 3.5 and ask it to generate an 8-bit frog svg, or send a pdf, and ask it to explain the contents.

### What browsers are you seeing the problem on?

Chrome

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment, marked as answer]
This is unique to your proxy. It works on most OpenAI-like providers and official Anthropic
